api/logbroker.proto

```diff
@@ -94,14 +94,14 @@ service Logs {
 }
 
 message SubscribeLogsRequest {
-	// LogSelector describes the logs to which the subscriber is 
+	// LogSelector describes the logs to which the subscriber is
 	LogSelector selector = 1;
 
 	LogSubscriptionOptions options = 2;
 }
 
 message SubscribeLogsMessage {
-	repeated LogMessage messages = 1 [(gogoproto.nullable) = false];
+	LogMessage message = 1 [(gogoproto.nullable) = false];
```

Comment:
It still seems useful for this to be a repeated field so we can send down multiple messages at a time if we later decide to. That will probably cut down some framing overhead.